splat-radio.sh

```diff
@@ -146,14 +227,37 @@ EOF
 # -d = path to elevation data files
 # -o = output file
 # -olditm = old ITM analysis, seems more accurate
-if [ $ERP -eq 0 ] ; then
-	time $NICE splat -t $QTHFILE -R 100 -L 6 -m 1.333 -db 160 -erp $ERP \
-		-d $TOPOFILEDIR -b $BORDERFILE -olditm -o $MAPPPMFILE
+# -ngs = display topograpy as white
+
+#add all needed options:
+#danger may not be compatible with spaces in filenames
+SPLAT_OPTIONS+=("-olditm")
+SPLAT_OPTIONS+=("-metric")
```

Comment:
What is the practical effect of this?  Just HAAT output changes or something else?